src/components/Editor/__tests__/EditorTabs.test.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { fireEvent, render } from "@testing-library/react";
+import EditorTabs from "../EditorTabs";
+
+describe("EditorTabs", () => {
+  it('should render two tabs with titles "JSX" and "CSS"', () => {
+    const { getByText } = render(<EditorTabs selectedTab="jsx" />);
+    expect(getByText("JSX")).toBeInTheDocument();
+    expect(getByText("CSS")).toBeInTheDocument();
+  });
+
+  it('should apply the "selected" prop to the currently selected tab', () => {
+    const { getByText } = render(<EditorTabs selectedTab="css" />);
+    expect(getByText("CSS")).toHaveClass("selected");
```

Comment:
Probably not for now but what about using `aria-label` instead?

Reply:
We should definitely do it to make the addon a11y, I want to try and use a native storybook component to get all styling and a11y for free, but they don't have this kind of component.
But again, that's definitely something we want on later